### DISCUSSION TITLE
support for underline emphasis

### DISCUSSION THREAD

**Discussion by metronome12** [gonzalezreal/swift-markdown-ui#191] [Q&A] [answered]
Underline text emphasis would be a great feature.

**Comment by gonzalezreal** [comment, marked as answer]
Which version of MarkdownUI are you using?

With MarkdownUI 2.x, you can achieve that by styling the Markdown view. For instance:

```swift
Markdown {
  "*This text is underlined and italicized!*"
}
.markdownTextStyle(\.emphasis) {
  FontStyle(.italic)
  UnderlineStyle(.single)
}
```

Check the [docs](https://swiftpackageindex.com/gonzalezreal/swift-markdown-ui/2.0.1/documentation/markdownui/textstyle) and the [Text Styles](https://github.com/gonzalezreal/swift-markdown-ui/blob/main/Examples/Demo/Demo/TextStylesView.swift) section in the demo project for more info :wink:.

**Reply by metronome12** [reply]
Ahh okay. I'd rather have the emphasis inline. Something like the '_fox_' so that it's treated as any other emphasis. 

But I also realize the difficulty of doing this and this is likely why it is not implemented in the Text() (swiftUI) object.
